client/src/components/menu/CommunityBtn.tsx

```diff
@@ -0,0 +1,193 @@
+import { FC, FunctionComponent, useMemo } from 'react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Button from '@material-ui/core/Button';
+import SvgIcon from '@material-ui/core/SvgIcon';
+import { makeStyles, Theme, Link } from '@material-ui/core';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import GitHubIcon from '@material-ui/icons/GitHub';
+import { ReactComponent as peopleIcon } from '../../assets/icons/people.svg';
+import { ReactComponent as slackIcon } from '../../assets/icons/slack.svg';
+import qrcodePath from '../../assets/imgs/wechat_qrcode.png';
+
+const slackLink = 'https://slack.milvus.io';
+const githubLink = 'https://github.com/milvus-io/milvus/discussions';
```

Comment:
Could you please change constant value to uppercase, like SLACK_LINK?
I think it will be more clearly.